include/dlaf/eigensolver/reduction_to_band/mc.h

```diff
@@ -532,6 +570,7 @@ pika::shared_future<common::internal::vector<T>> computePanelReflectors(
 
   auto panel_tiles = pika::when_all(matrix::select(mat_a, ai_panel_range));
 
+  // TODO: This needs a when_all equivalent that can take a range of senders.
   return pika::dataflow(getHpExecutor<Backend::MC>(), std::move(panel_task), std::move(panel_tiles),
```

Comment:
@albestro this is the last of the places that still requires dataflow for a vector of futures.